docs/site/BelongsTo-relation.md

```diff
@@ -210,9 +213,18 @@ export class CategoryRepository extends DefaultCrudRepository<
   Category,
   typeof Category.prototype.id
 > {
-  public readonly parent: BelongsToAccessor<Category, number>;
+  public readonly parent: BelongsToAccessor<Category, typeof Category.prototype.id>;
+  public readonly categories: HasManyRepositoryFactory<
+    Category,
+    typeof Category.prototype.id
+  >;
+
   constructor(@inject('datasources.db') dataSource: DbDataSource) {
     super(Category, dataSource);
+    this.categories = this.createHasManyRepositoryFactoryFor(
+      'categories',
+      Getter.fromValue(this),
+    );
     this.parent = this._createBelongsToAccessorFor(
```

Comment:
I know this isn't related to your change, but can you update this line to use `createBelongsToAccessorFor` instead of `_createBelongsToAccessorFor`?

Reply:
good catch! fixed